capa/features/extractors/dnfile/types.py

```diff
@@ -42,15 +42,19 @@ def __repr__(self):
         return str(self)
 
     @staticmethod
-    def format_name(class_: str, namespace: str = "", member: str = ""):
+    def format_name(class_: tuple, namespace: str = "", member: str = ""):
+        if type(class_) == tuple:
```

Comment:
Do we expect `class_` to ever *not* be a `tuple`?

Reply:
Corrected to properly treat incoming data as always being a tuple